app/services/twilio_service.rb

```diff
@@ -35,6 +35,21 @@ def from_number
     "+1#{account['number']}"
   end
 
+  def valid_number?(to_number)
+    to_number = format_phone_number(to_number)
+    begin
+      response = client.lookups.phone_numbers(to_number).fetch
```

Comment:
Ok so I think the "lookup" API is the wrong API for us to use here, we need to actually attempt the SMS or voice call in the foreground.

I looked up a few of the phone numbers from our most recent NewRelic exceptions that failed (+41 is Switzerland, failing currently in production for example) and they passed using the lookups API so this is not sufficient IMO. Let me know if you have any questions about what I mean.